modules/ossm-running-v2-v3-cluster-wide-deployment-model.adoc

```diff
@@ -0,0 +1,150 @@
+
+In {SMProduct} 2.6, you can check your deployment model from the `ServiceMeshControlPlane` under `spec.mode`:
+
+.Example `ServiceMeshControlPlane` yaml
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  mode: ClusterWide
+----
+
+To prevent conflicts with {SMProduct} 3.0 when using the {SMProduct} 2.6 cluster-wide deployment model, you need to configure the `ServiceMeshControlPlane` resource to restrict namespaces to only those belonging to (SMProduct) 2.6.
+
+//NOTES: 10/23/2024
+
+//DO NOT USE VERSION NUMBER ATTRIBUTE. It is set for 3.0, will likely change, and it is unclear if later versions would impact this procedure for moving from 2.6.
+//Only moving from 2.6 has been tested and verified by QE.
+//Cluster-wide requires changes to OSSM 2.6 ServiceMeshControlPlane before installing OSSM 3.0
+
+.Prerequisites
+
+* You are running {ocp-product-title} 4.14 or later.
+* You are running {SMProduct} 2.6.
+
+If you are not running {SMProduct} 2.6, you must upgrade to 2.6 before following this procedure. To upgrade from you current {SMProduct} version to 2.6, see: link:https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/upgrading-ossm.html[Upgrading Service Mesh 2.x]
+
+.Procedure
+
+. Configure `discoverySelectors`, and set the `ENABLE_ENHANCED_RESOURCE_SCOPING` environment variable on the pilot container to `true` in your {SMProduct} 2.6 `ServiceMeshControlPlane` custom resource (CR):
++
+.Example `ServiceMeshControlPlane` CR
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.6
+  mode: ClusterWide
+  meshConfig:
+    discoverySelectors:
+      - matchExpressions:
+        - key: maistra.io/member-of
+          operator: Exists
+  runtime:
+    components:
+      pilot:
+        container:
+          env:
+            ENABLE_ENHANCED_RESOURCE_SCOPING: 'true'
+----
+
+. Install the {SMProduct} 3 Operator.
+
+. Create an `IstioCNI` resource in the `istio-cni` namespace.
+
+. Create an `Istio` resource in a different `namespace` than the `namespace` used in the `ServiceMeshControlPlane` resource in {SMProduct} 2.6. In this example, `istio-system3` is being used:
++
+.Example `Istio` resource with `istio-system3`
+[source, yaml]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: ossm3 # <1>
+spec:
+  namespace: istio-system3 # <2>
+  values:
+    meshConfig:
+      discoverySelectors: # <3>
+        - matchExpressions:
+          - key: maistra.io/member-of
+            operator: DoesNotExist
+  updateStrategy:
+    type: InPlace
+  version: v1.23.0
+----
+<1> Do not use `default` as the name.
+<2> Must be different from the `namespace` used in the `ServiceMeshControlPlane` resource in {SMProduct} 2.6.
+<3> Use `discoverySelectors` to ignore {SMProduct} 2.6 namespaces. All other namespaces will be part of the {SmProduct} 3.0 mesh.
```

Comment:
As a rule, say the "why" before the "what". So change to: "To ignore...., use ...".